src/viam/sdk/rpc/dial.hpp

```diff
@@ -63,7 +64,11 @@ class DialOptions {
 
     /// @brief Allows the RPC connection to be downgraded to an insecure connection if detected.
     /// This is only used when credentials are not present.
-    bool allow_insecure_downgrade_;
+    bool allow_insecure_downgrade_ = false;
+
+    /// @brief Duration before the dial connection times out
+    /// Set to 20sec to match _defaultOfferDeadline in goutils/rpc/wrtc_call_queue.go
+    std::chrono::seconds timeout_ = std::chrono::seconds(20);
```

Comment:
I think you can even just write this like `std::chrono::seconds timeout_{20};`

Reply:
Yep!